client/jetpack-cloud/sections/agency-dashboard/header/index.tsx

```diff
@@ -0,0 +1,28 @@
+import config from '@automattic/calypso-config';
+import classNames from 'classnames';
+import { useTranslate } from 'i18n-calypso';
+import FormattedHeader from 'calypso/components/formatted-header';
+import OlarkChat from 'calypso/components/olark-chat';
+import JetpackComMasterbar from 'calypso/jetpack-cloud/sections/pricing/jpcom-masterbar';
+import { preventWidows } from 'calypso/lib/formatting';
+import './style.scss';
+
+export default function Header() {
+	const identity = String( config( 'olark_chat_identity' ) );
+	const translate = useTranslate();
+	const title = translate( 'Security, performance, and growth tools for WordPress' );
+
+	return (
+		<>
+			{ identity && <OlarkChat identity={ identity } /> }
+			<JetpackComMasterbar />
+			<div className={ classNames( 'header' ) }>
+				<FormattedHeader
+					className="header__main-title"
+					headerText={ preventWidows( title ) }
+					align="center"
+				/>
+			</div>
+		</>
+	);
+}
```

Comment:
I see we already have a component that looks exactly like this (see https://github.com/Automattic/wp-calypso/blob/49b24160513c22585c26e66b39bcbeff74a62fa7/client/jetpack-cloud/sections/partner-portal/header/index.tsx#L10). Can we use the same in both sections? I suspect we would have to move the component up the hierarchy to a place that would make sense.